packages/app-store/routing-forms/components/react-awesome-query-builder/config/config.tsx

```diff
@@ -1,11 +1,19 @@
 import type { ChangeEvent } from "react";
-import type { Settings, Widgets, SelectWidgetProps, SelectWidget } from "react-awesome-query-builder";
+import type {
+  Settings,
+  Widgets,
+  SelectWidgetProps,
+  SelectWidget as SelectWidgetType,
+} from "react-awesome-query-builder";
+
+import { EmailField as EmailWidget } from "@calcom/ui";
```

Comment:
Not importing EmailField in widgets.tsx now as somehow that is causing circular dependency(only for website repo)